pycqed/measurement/detector_functions.py

```diff
@@ -2368,3 +2368,34 @@ def get_values(self):
     def finish(self):
         if self.AWG is not None:
             self.AWG.stop()
+
+class Function_Detector_list(Soft_Detector):
```

Comment:
I remember deleting this in favor of the more generic Function_Detector (that works both for lists and dict returning functions).